src/patternfly/components/TreeView/tree-view.scss

```
@@ -5,9 +5,8 @@ $pf-c-tree-view--MaxNesting: 10;
   --pf-c-tree-view--PaddingBottom: var(--pf-global--spacer--sm);
 
   // Link base
-  --pf-c-tree-view__node--indent--base: var(--pf-global--spacer--sm);
-  --pf-c-tree-view__node--nested-indent--base: var(--pf-global--spacer--lg);
-  --pf-c-tree-view__item--m-expandable__node--nested-indent--base: var(--pf-global--spacer--lg);
+  --pf-c-tree-view__node--indent--base: calc(var(--pf-global--spacer--lg) + var(--pf-global--spacer--sm));
+  --pf-c-tree-view__node--nested-indent--base: calc(var(--pf-global--spacer--lg) + var(--pf-global--spacer--sm));
```

```suggestion
  --pf-c-tree-view__node--nested-indent--base: var(--pf-global--spacer--lg);
```

Looks like this has a bit too much space?